spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -458,6 +458,9 @@ public void testSinkDestinationWithTwoApps() throws Exception {
 	public void testDestinationsOnBothSides() throws Exception {
 		assertEquals(0, repository.count());
 		String definition = ":bar > filter > :foo";
+		AppStatus status = mock(AppStatus.class);
```

Comment:
Not sure how does this change helps testing the return of `stream` and `task`definition resources on `save` operations.